src/app/components/personalised-homepage-components/ItemCard.js

```diff
@@ -0,0 +1,18 @@
+'use client'
+import { ReactNode } from "react";
+
+function ItemCard({ children, title, img, trackID, setPlaybackID }) {
+
+    return (
+        <div className="hover:bg-gray-700 focus:bg-gray-700 active:bg-gray-700 delay-50 duration-100 bg-ablack p-5 rounded-lg min-w-[10vw] group p-2 m-2" id={trackID} onClick={setPlaybackID}>
+            <img src={img}
```

Comment:
Results from my linters. No action required:
- 'ReactNode' is declared but its value is never read
- Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element

Whenever I have time for it, I'll try and implement these checks whenever we push changes to GitHub